e2e/testdata/help/oci-attach.txt

```diff
@@ -0,0 +1,17 @@
+Attach console to a running container process (root user only)
```

Comment:
there should be a README.md here explaining:

- how to create these files (the output of `singularity help` is sensitive to some environment variables)
- how to name these files
- how / when to update these files

Reply:
Done!